aws/src/test/java/aws/cloudformation/AwsTestConstants.java

```diff
@@ -8,7 +8,7 @@ public class AwsTestConstants {
     public static final String instanceName = "${non.existing:eapSunstoneInjectInstance}";
     // must be same as in sunstone.properties
     public static final String region = "${ec2.region:us-east-1}";
-    public static final String eapMngmtUser = "${non.existing:admin}";
-    public static final String eapMngmtPassword = "${non.existing:pass.1234}";
-    public static final String eapMngmtPort = "${non.existing:9990}";
+    public static final String wildflyMngmtUser = "${non.existing:admin}";
+    public static final String wildflyMngmtPassword = "${non.existing:pass.1234}";
+    public static final String wildflyMngmtPort = "${non.existing:9990}";
```

Comment:
Could it be just `MngmtUser` and so on? I think it should be ok considering that people are used to it from Arquillian.

Reply:
Sure.